app/kubemci/pkg/kubeutils/utils.go

```diff
@@ -23,6 +23,16 @@ import (
 	multierror "github.com/hashicorp/go-multierror"
 	kubeclient "k8s.io/client-go/kubernetes"
 	"k8s.io/client-go/tools/clientcmd"
+	"sort"
+	"k8s.io/client-go/tools/cache"
+	"k8s.io/apimachinery/pkg/util/intstr"
```

Comment:
General FYI about ordering of imports.
We keep 3 sections: standard libs; non-standard libs outside this repo; libs from this repo.
sections are separated using a line break and imports are sorted within each section.